test/step-definitions/cityWeatherPageSteps.js

```diff
@@ -0,0 +1,15 @@
+import { When, Then } from "@wdio/cucumber-framework";
+import cityWeatherPage from "../pageobjects/cityWeatherPage.js";
+import { assert } from 'chai';
+
+When(/^I go to the city weather page$/, async () => {
```

Comment:
This strange step. it has name "When" and "I go to city...." but in body I don't see any action. Maybe would be better change this step on then change strinf "I am on specific weather page"  and use assertion inside

Reply:
Resolved